docs/reference-architectures/data/enterprise-bi-sqldw.md

```diff
@@ -0,0 +1,402 @@
+
+### Authorization
+
+Azure Analysis Services uses Azure Active Directory (Azure AD) to authenticate users who connect to an Analysis Services server. You can restrict what data a particular user is able to view, by creating roles and then assigning Azure AD users or groups to those roles. For each role, you can: 
+
+- Protect tables or individual columns. 
+- Protect individual rows based on filter expressions. 
+
+For more information, see [Manage database roles and users](/azure/analysis-services/analysis-services-database-users).
+
+## Deploy the solution
+
+A deployment for this reference architecture is available on [GitHub][ref-arch-repo-folder]. It deploys the following:
+
+  * A Windows VM to simulate an on-premises database server. It includes SQL Server 2017 and related tools, along with Power BI Desktop.
+  * An Azure storage account that provides Blob storage to hold data exported from the SQL Server database.
+  * An Azure SQL Data Warehouse instance.
+  * An Azure Analysis Services instance.
+
+### Prerequisites
+
+1. Clone, fork, or download the zip file for the [Azure reference architectures][ref-arch-repo] GitHub repository.
+
+2. Install the [Azure Building Blocks][azbb-wiki] (azbb).
+
+3. From a command prompt, bash prompt, or PowerShell prompt, login to your Azure account by using the command below and following the instructions.
+
+  ```bash
+  az login  
+  ```
+
+### Deploy the simulated on-premises server
+
+First you'll deploy a VM as a simulated on-premises server, which includes SQL Server 2017 and related tools. This step also loads the sample [Wide World Importers OLTP database](/sql/sample/world-wide-importers/wide-world-importers-oltp-database) into SQL Server.
+
+1. Navigate to the `data\enterprise-bi-sqldw\onprem\templates` folder of the repository you downloaded in the prerequisites above.
+
+2. In the `onprem.parameters.json` file, replace the values for `adminUsername` and `adminPassword`. Also change the values in the `SqlUserCredentials` section to match the user name and password. Note the `.\\` prefix in the userName property.
+    
+    ```bash
+    "SqlUserCredentials": {
+      "userName": ".\\username",
+      "password": "password"
+    }
+    ```
+
+3. Run `azbb` as shown below to deploy the on-premises server.
+
+    ```bash
+    azbb -s <subscription_id> -g <resource_group_name> -l <location> -p onprem.parameters.json --deploy
+    ```
+
+4. The deployment may take 20 to 30 minutes to complete, which includes running the [Desired State Configuration](/powershell/dsc/overview) (DSC) to install the tools and restore the database. Verify the deployment in the Azure portal by reviewing the resources in the resource group. You should see the `sql-vm1` virtual machine and its associated resources.
```

Comment:
Change to "a DSC script", i.e.  ->
"includes running a [Desired State Configuration](/powershell/dsc/overview) (DSC) script to install"